File: 1-modelagem-codigo-php/src/app/Produto.php
<?php

namespace App;

class Produto
{
    private int $id;
    private string $nome;
    private float $preco;

    public function __construct(int $id, string $nome, float $preco)
    {
        $this->id = $id;
        $this->nome = $nome;
        $this->preco = $preco;
    }

    public function getId(): int
    {
        return $this->id;
    }

    public function getNome(): string
    {
        return $this->nome;
    }

    public function getPreco(): float
    {
        return $this->preco;
    }
}

File: 1-modelagem-codigo-python/app/produto.py
class Produto:
    def __init__(self, id: int, nome: str, preco: float):
        self.__id = id
        self.__nome = nome
        self.__preco = preco

    def get_id(self) -> int:
        return self.__id

    def get_nome(self) -> str:
        return self.__nome

    def get_preco(self) -> float:
        return self.__preco